### DISCUSSION TITLE
get owner id to show in RelationManager

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#1848] [Help]
Hi, how can i get owner id to show in RelationManager?

**Comment by danharrin** [comment]
Show where?

**Reply by jcgmalave** [reply]
I have a Product Resource and has many ProductDetail, via has many relationship, I need to to show the product id in the modal with the form to create Product Details associsted to Products

**Reply by danharrin** [reply]
If you're using a relation manager, the details will save to the relationship without you needing to access the ID of the product. Have you tried it?

**Reply by jcgmalave** [reply]
Yes i am able to save te record, but I need to show the id of the product in the modal, for example, the product 1 has detallado 1, 2, 3 & 4, in the table of the relation manager, I need to show the id 1 of the product

**Reply by danharrin** [reply]
`Placeholder::make('ownerId')->content(fn ($livewire) => $livewire->ownerRecord->getKey())` maybe?

**Reply by jcgmalave** [reply]
Thanks a Lot :)
